.github/workflows/deploy-env.yaml

```diff
@@ -0,0 +1,48 @@
+name: Deploy Environment
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to'
+        type: choice
+        required: true
+        default: 'ops-preview-1'
+        options:
+        options:
+        - ops-preview-1
+      region:
+        description: 'Region containing cluster to deploy to'
+        type: choice
+        required: true
+        default: 'us-east-1'
+        options:
+        - us-east-1
+        - us-east-2
+        - eu-central-1
+
+jobs:
+  deploy:
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📥 Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install QEMU
+        run: sudo apt-get install -y qemu-user-static
+
+      - name: 🧰 Setup Nix
+        uses: cachix/install-nix-action@v21
+        with:
+          extra_nix_config: |
+            system = aarch64-linux
+
+      - name: 🚀 Deploy
+        run: |
+          echo "${{ secrets.ENVRC }}" > .envrc
+          source .envrc
+          nix develop
```

Comment:
This will not provide you with `aws` executable if you don't do `-c`

Reply:
It provides `aws` executable without the flag.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/actions/runs/10089733209/job/27897743167

Reply:
I don't think so either, because `nix develop` drops you into a new bash. If it's not an interactive session (like here), it will just exit immediately with code 0.

If `aws` is available, it means it was previously on `PATH`, probably added there by GitHub as part of standard runner toolbox?

Can you change:

```sh
nix develop
```

to:

```sh
which aws || true
nix develop
which aws
```

Then we'll see if `aws` existed before `nix develop`, and whether the second path is in `/nix/store/…-aws/bin/aws` or provided by GitHub.

In the latter case you can do this (like `std.yml`), for reproducibility:

```sh
nix develop -L --command bash -c '
  set -euo pipefail
  echo "yes" | nix run .#cardano-services.${{ inputs.environment }}.apply
'
```